src/csharp/Microsoft.Spark.Worker.UnitTest/TaskRunnerTests.cs

```diff
@@ -26,36 +27,37 @@ public void TestTaskRunner()
             var taskRunner = new TaskRunner(0, clientSocket, false, payloadWriter.Version);
             Task clientTask = Task.Run(() => taskRunner.Run());
 
-            TestTaskRunnerReadWrite(serverListener, payloadWriter);
+            ISocketWrapper serverSocket = serverListener.Accept();
 
+            TestTaskRunnerReadWrite(serverSocket, payloadWriter);
+
+            serverSocket.Dispose();
             Assert.True(clientTask.Wait(5000));
         }
 
         internal static void TestTaskRunnerReadWrite(
-            ISocketWrapper serverListener,
+            ISocketWrapper serverSocket,
             PayloadWriter payloadWriter)
         {
-            using (ISocketWrapper serverSocket = serverListener.Accept())
+            System.IO.Stream inputStream = serverSocket.InputStream;
+            System.IO.Stream outputStream = serverSocket.OutputStream;
+
+            payloadWriter.WriteTestData(outputStream);
+            // Now process the bytes flowing in from the client.
+            List<object[]> rowsReceived = PayloadReader.Read(inputStream);
+
+            // Validate rows received.
+            Assert.Equal(10, rowsReceived.Count);
+            for (int i = 0; i < 10; ++i)
             {
-                System.IO.Stream inputStream = serverSocket.InputStream;
-                System.IO.Stream outputStream = serverSocket.OutputStream;
-
-                payloadWriter.WriteTestData(outputStream);
-                // Now process the bytes flowing in from the client.
-                List<object[]> rowsReceived = PayloadReader.Read(inputStream);
-
-                // Validate rows received.
-                Assert.Equal(10, rowsReceived.Count);
-                for (int i = 0; i < 10; ++i)
-                {
-                    // Two UDFs registered, thus expecting two columns.
-                    // Refer to TestData.GetDefaultCommandPayload().
-                    object[] row = rowsReceived[i];
-                    Assert.Equal(2, rowsReceived[i].Length);
-                    Assert.Equal($"udf2 udf1 {i}", row[0]);
-                    Assert.Equal(i + i, row[1]);
-                }
+                // Two UDFs registered, thus expecting two columns.
+                // Refer to TestData.GetDefaultCommandPayload().
+                object[] row = rowsReceived[i];
+                Assert.Equal(2, rowsReceived[i].Length);
+                Assert.Equal($"udf2 udf1 {i}", row[0]);
+                Assert.Equal(i + i, row[1]);
             }
         }
     }
+    
```

Comment:
newline added?

Reply:
Looks like my Visual studio is adding newlines to some files on its own. Removed.